github3/repos/repo.py

```diff
@@ -687,14 +687,15 @@ def create_commit(self, message, tree, parents, author=None,
         return self._instance_or_null(Commit, json)
 
     @requires_auth
-    def create_deployment(self, ref, force=False, payload='',
+    def create_deployment(self, ref, required_contexts=[], payload='',
                           auto_merge=False, description='', environment=None):
         """Create a deployment.
 
         :param str ref: (required), The ref to deploy. This can be a branch,
             tag, or sha.
-        :param bool force: Optional parameter to bypass any ahead/behind
-            checks or commit status checks. Default: False
+        :param list required_contexts Optional array of status contexts
```

Comment:
The docs job is failing because you need this to be

```
:param list required_contexts: Optional array ...
```

Not 

```
:param list required_contexts Original array
```

(Notice the `:` after `required_contexts`)